packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -929,6 +917,81 @@ class StripeHelper {
     };
   }
 
+  /**
+   * Extract source details for billing emails
+   *
+   * @param {Source | Card} source
+   */
+  async extractSourceDetailsForEmail(source) {
+    if (source.object !== 'card') {
+      // We shouldn't get here - all sources should currently be cards.
+      throw error.internalValidationError(
+        'extractSourceDetailsForEmail',
+        source,
+        new Error(`Payment source was not card: ${source.id}`)
+      );
+    }
+
+    const customer = await this.expandResource(source.customer, 'customers');
+    if (customer.deleted === true) {
+      throw error.unknownCustomer(source.customer);
+    }
+
+    // Follow subhub's lead and just use the customer's first active
+    // subscription as source of plan & product data.
```

Comment:
How does this affect multiple subscriptions?

Reply:
It affects multiple subscriptions insofar as it ignores them. It just uses the first active subscription as the one to mention in the email. This is our current behavior through subhub,